agent/mitm/handlers/HttpUpgradeHandler.ts

```diff
@@ -96,13 +96,16 @@ export default class HttpUpgradeHandler extends BaseHttpHandler {
     for (let i = 0; i < serverResponse.rawHeaders.length; i += 2) {
       responseMessage += `${serverResponse.rawHeaders[i]}: ${serverResponse.rawHeaders[i + 1]}\r\n`;
     }
+    this.context.resourceBodySize = responseMessage.length + 2; // 2 for \r\n
```

Comment:
I'm not sure what to do about this one either. The body is empty for an http upgrade response, I believe. I do not think we should be adding up header string sizes.